packages/configuration-backend/src/service.ts

```diff
@@ -233,6 +266,62 @@ export const getMerchantModels = async (
   }
 }
 
+function getAllowedKeysToMerchant() {
+  return [
+    'merchantName',
+    'merchantStreet',
+    'merchantZip',
+    'merchantCity',
+    'merchantEmail',
+    'merchantPhone',
+    'merchantUrl',
+    'merchantTermsOfServiceUrl',
+    'merchantBusinessId',
+  ]
+}
+
+export const getMerchantValues = async (
+  namespace: string,
+  merchantId: string | null
```

Comment:
This function throws error, if merchantId is null. Then should the type of merchantId be only type of string and then in the callers end perform this function call only if the merchantId is present?

Reply:
I was thinking to do try catch for exceptions, in controller to prevent multiple check conditions for merchantId exists before function call. Because guard clauses in function can be catched easier than in controller because error does not allways mean that we need to stop processing.

Reply:
Makes perfect sense, no need to change this.